src/css/override_components.scss

```diff
@@ -183,3 +159,16 @@ input:focus {
     width: $grid-4;
   }
 }
+
+.fa {
```

Comment:
This should be removed — I was trying something that I don't think is going anywhere!